pkg/controller/proxyconfig/proxyconfig_controller.go

```diff
@@ -0,0 +1,172 @@
+package proxyconfig
+
+import (
+	"context"
+
+	configv1 "github.com/openshift/api/config/v1"
+	loggingv1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	logforwarding "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
+	"github.com/openshift/cluster-logging-operator/pkg/k8shandler"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+var log = logf.Log.WithName("controller_proxyconfig")
+
+const (
```

Comment:
i'm wondering if we should actually move these from this controller to a different location like the types file since we use at least some of these values in multiple places...
cc @jcantrill

Reply:
> i'm wondering if we should actually move these from this controller to a different location like the types file since we use at least some of these values in multiple places...
> cc @jcantrill

Where such types.go is supposed to locate?  Something like one of these?
`cluster-logging-operator/types/types.go`
or
`cluster-logging-operator/pkg/types/types.go`

I created `pkg/k8shandler/constants.go` in this PR.  I assume the file should be merged into the new "types.go", as well...

Reply:
sorry, i meant this file https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1/clusterlogging_types.go

Reply:
they are constants so maybe something more obvious like `cluster-logging-operator/pkg/constants/constants.go`